PlayerControls/sources/AirPlayView.swift

```diff
@@ -35,22 +35,38 @@ public final class AirPlayView: UIView {
     var props: Props = Props.empty {
         didSet { setNeedsLayout() }
     }
+    
+    #if (arch(i386) || arch(x86_64)) && os(iOS)
```

Comment:
What is the purpose of these checks?

Reply:
These checks are to determine if we're launching this project on a device or on a Simulator. Because MPVolumeView still doesn't works stable and it may simply not appear on the screen, I've decided to resolve the problem directly in this way.

Reply:
What? What do you mean - `doesnot work stable`?

Reply:
By unstable behaivor I mean the one that when you laucnh our Demo on Simulator with AirPlay enabled, it renderes emty AirPlayView without MPVolumeView. The reason why it is happening was described in #48 PR.